tests/check_imagestreams.sh

```diff
@@ -3,7 +3,12 @@
 set -x
 
 check_imagestreams=$(dirname "$(readlink -f "$0")")/../check_imagestreams.py
+show_all_imagestreams=$(dirname "$(readlink -f "$0")")/../show_all_imagestreams.py
 "${PYTHON-python3}" "$check_imagestreams" "2.5"
 test $? -eq 1
 "${PYTHON-python3}" "$check_imagestreams" "2.4"
 test $? -eq 0
+echo "Check full is output"
```

Comment:
Could you please rephrase this string? I am not sure what it means.

Reply:
Fixed by https://github.com/sclorg/container-common-scripts/pull/372/commits/1903527ebf5564ecb8ca5b9c155dbeb770a76d08. Good point.